Makefile

```diff
@@ -175,5 +188,37 @@ test-e2e: generate fmt vet manifests
 	test -f $(ENVTEST_ASSETS_DIR)/setup-envtest.sh || curl -sSLo $(ENVTEST_ASSETS_DIR)/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.6.3/hack/setup-envtest.sh
 	source ${ENVTEST_ASSETS_DIR}/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR); go test $(TEST_E2E_PKGS) -coverprofile cover.out -ginkgo.v -ginkgo.progress -v
 
+.PHONY: bundle-validate
 bundle-validate:
 	$(OPERATOR_SDK) bundle validate ./bundle
+
+.PHONY: bundle-validate-image
+bundle-validate-image:
+	$(OPERATOR_SDK) bundle validate $(BUNDLE_IMG)
+
+.PHONY: bundle-custom-updates
+bundle-custom-updates: BUNDLE_PREFIX=dev$(CURRENT_DATE)
+bundle-custom-updates:
+ifndef YQ
```

Comment:
I'm not sure whether bundle-custom-updates task and related content should be added in another PR.

In any case I'll leave some feedback:
* Can we create a Makefile task that gathers 'yq' and leaves the binary in $(PROJECT_PATH)/bin as with other commands? it seems there's a URL available to download yq binary so it should not be very complex.
* Existing `VERSION` Makefile variable can be used to specify the CSV version part in the CSV name

Reply:
Has this been tried? I see a number has been used as part of the custom bundle prefix. Are all modified fields compatible with numbers?

Reply:
* IMO, a Makefile target to gather `yq` adds complexity no required. `yq` can be installed from binaries and or package managers. Besides, the implementation needs to read system's arch and download required binary. Like `license-finder`, it is requirement in the developers environment. The Makefile will warn if not installed. Everything can be done, but the effort does not pay off. Feel free to try it.

* `VERSION` var used

* I have tried and works like a charm. Try it!